### PR TITLE
Additional Tenable IO Commands

### PR DESCRIPTION
**Pull request by chrisbalmer**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added the following commands to the tenable IO integration:

- Added command `tenable-io-list-templates`
- Added command `tenable-io-get-plugin-details`
- Added command `tenable-io-list-policies`
- Added command `tenable-io-list-folders`
- Added command `tenable-io-list-plugins-by-family`
- Added command `tenable-io-list-plugin-families`
- Added command `tenable-io-list-scanners`
- Added command `tenable-io-create-scan`
- Added command `tenable-io-list-connectors`
- Added command `tenable-io-list-users`
- Added command `tenable-io-delete-user`

### CODE REVIEW

**Comment by jlevypaloalto** [comment]
Hi @chrisbalmer, we haven’t heard from you in a while.
Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

**Comment by jlevypaloalto** [comment]
Hi @chrisbalmer, 
Thank you for your contribution. 
I have not received a response from you in the last two weeks, so I will close your PR now.
Please feel free to re-open it when you are available to continue. 
Thanks again.

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 1]
```diff
@@ -173,6 +175,55 @@ def list_scan_filters(self):
...
+        return self._http_request(
+            'GET', f'plugins/families/{family_id}')
+
+    def list_scanners(self):
+        return self._http_request(
+            'GET', f'scanners')
+
+    def list_connectors(self, limit, offset, sort):
+        url = "settings/connectors"
+        query = ""
```
[the remark this answers is not in the document]
Comment (chrisbalmer) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:204]: fixed

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 2]
```diff
@@ -555,6 +612,25 @@ def send_asset_details_request(asset_id: str) -> Dict[str, Any]:
     return res.json()
 
 
+def send_plugin_details_request(plugin_id: str) -> Dict[str, Any]:
+    """Gets plugin details using the '{BASE_URL}plugins/plugin/{plugin_id}' endpoint.
```
[the remark this answers is not in the document]
Comment (chrisbalmer) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:616]: moved to the client class

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 3]
```diff
@@ -1657,6 +1733,195 @@ def get_audit_logs_command(client: Client, from_date: Optional[str] = None, to_d
...
 
+def list_policies_command(client: Client) -> CommandResults:
+    response_dict = client.list_policies()
+    readable_output = tableToMarkdown(
+        f'Tenable Policies', response_dict.get("policies"), headers=["id", "name", "creation_date", "last_modification_date", "visibility", "shared", "owner", "owner_id", "description", "template_uuid", "no_target","user_permissions"])
+    return CommandResults(
+        outputs_prefix='TenableIO.Policies',
+        outputs=response_dict.get("policies"),
+        readable_output=readable_output,
+        raw_response=response_dict.get("policies"))
```
[the remark this answers is not in the document]
Comment (chrisbalmer) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:1744]: fixed all raw_response values

**Pull request closed without merging**